tests/test_codebuild_cli.py

```diff
@@ -0,0 +1,204 @@
+import json
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+import boto3
+import pytest
+from click.testing import CliRunner
+from moto import mock_codebuild
+from moto import mock_iam
+from moto import mock_ssm
+from moto import mock_sts
+
+from commands.codebuild_cli import AWS_REGION, codedeploy, create_codedeploy_role, slackcreds
+from commands.codebuild_cli import check_git_url, link_github
+from commands.codebuild_cli import check_service_role
+from commands.codebuild_cli import update_parameter
+from utils.aws import check_aws_conn
+
+@pytest.fixture
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    moto_credentials_file_path = Path(__file__).parent.absolute() / 'dummy_aws_credentials'
+    os.environ['AWS_SHARED_CREDENTIALS_FILE'] = str(moto_credentials_file_path)
+
+
+@pytest.fixture
+def alias_session(aws_credentials):
+    with mock_iam():
+        session = boto3.session.Session()
+        session.client("iam").create_account_alias(AccountAlias="foo")
+        
+        yield session
+
+
+# Not much value in testing these while moto doesn't support `import_source_credentials`` or `list_source_credentials`
+# def test_import_pat():
+#     ...
+# def test_check_github_conn():
+#     ...
+
+
+@mock_iam
+def test_check_service_role_does_not_exist(capfd, aws_credentials):
+    """Test that check_service_role outputs expects error method when codebuild role does not exist."""
```

Comment:
Can you explain the purpose of this string?

It looks to me like it might be kind of a comment, but done as a string which is not output.

Reply:
I'm using docstrings to make it more explicit what each test is actually testing. This one unhelpfully contains a typo, however, making it less useful...

Reply:
This is known as a Docstring. Moar info here: https://peps.python.org/pep-0257/#what-is-a-docstring